pna.p4

```diff
@@ -444,7 +444,7 @@ enum PNA_CounterType_t {
 
 extern Counter<W, S> {
   Counter(bit<32> n_counters, PNA_CounterType_t type);
-  void count(in S index);
+  void count(in S index, @optional in bit<32> increment);
```

Comment:
Is there a reason why you use @optional here when doing the following should also work:
```
void count(in S index);
void count(in S index, in bit<32> increment);
```
I don't see why @optional is needed when you can do it similarly via overloading.